atlp-rwanda/Avengers-Phantom-FE
# fixed login form

## What does this PR do?

Fix Login, logout, and Routes Protections

## Description of Task to be completed?

- [x] Implemented routed routes protection
- [x] Fixed logout
- [x] Fixed Login 
- [x] Implemented Routes Protection
- [x] Implemented dynamic username updates on the dashboard

Done


### How should this be manually tested?

step1: clone git repo if not yet done
step2: cd Avengers-Phantom-FE
step3: npm i or npm install to install dependencies
step4: `npm run dev` to start the server
step5: switch to` bg-fixing-routes-login-and-logout` branch
step6: Login to go to the dashboard with different user accounts to check Routes protection.
